tasks/bump.js

```diff
@@ -176,7 +186,7 @@ module.exports = function(grunt) {
 
 
     // PUSH CHANGES
-    runIf(opts.push, function() {
+    runIf( opts.push, function() {
```

Comment:
it adds noise to the commit to have things like this added in. commits should be small and clean